frontend/wallet/src/main/App.re

```diff
@@ -40,6 +40,14 @@ let createTray = (_settingsOrError, dispatch, wallets) => {
   let suffixItems =
     Menu.Item.[
       make(Separator, ()),
+      make(
```

Comment:
I'm intentionally leaving this in for now

Reply:
good idea